src/css/components/loading.scss

```diff
@@ -84,3 +84,12 @@ $indicator-size: 3.5rem;
     opacity: .2;
   }
 }
+
+.saving.saving-relative {
```

Comment:
@rememberlenny There should be a space between `.saving` and `.saving-relative`, yeah?